Sources/Build/BuildDelegate.swift

```diff
@@ -186,6 +186,104 @@ extension SPMLLBuild.Diagnostic: DiagnosticDataConvertible {
     }
 }
 
+class LinuxMainCommand: ExternalCommand {
```

Comment:
Please remove all occurrences of "Linux" from the names of these things -- there's nothing inherently Linux-specific about this so it should be named something like "TestMainCommand", "test-main-tool", etc.